Delicious/Application/Scenes/Main/MainViewController.swift

```diff
@@ -32,20 +34,26 @@ final class MainViewController: UIViewController, BindableType {
     }
     
     func bindViewModel() {
-        let requestTrigger = requestTriggerSubject
-            .startWith(("chicken", 10))
+        let searchTrigger = searchButton.rx.controlEvent(.touchUpInside)
```

Comment:
```suggestion
        let searchTrigger = searchButton.rx.tap.asDriver()
```
cái này ngắn viết luôn vào Input ko cần tách thành biến riêng như này anh nhé